Core/Rewriters/MockInstantiationRewriter.cs

```diff
@@ -62,13 +64,20 @@ public MockInstantiationRewriter (IFormatter formatter)
       }
 
       var nodeWithRewrittenDescendants = (InvocationExpressionSyntax) base.VisitInvocationExpression (node)!;
+      if (methodSymbol.Name == c_generatePartialMock && node.IsChildOfLocalDeclarationStatement())
+      {
+        return node;
+      }
```

Comment:
The 2 rewriters can be merged to reduce duplication. The rule concerning partial mocks also applies for local decl.